packages/mobile/src/identity/privacy.ts

```diff
@@ -97,20 +112,30 @@ interface SignMessageResponse {
 
 // Send the blinded message off to the phone number privacy service and
 // get back the theshold signed blinded message
-async function postToSignMessage(base64BlindedMessage: string, account: string) {
+async function postToSignMessage(
+  base64BlindedMessage: string,
+  account: string,
+  hashedPhoneNumber: string,
+  contractKit: ContractKit
+) {
   Logger.debug(`${TAG}@postToSignMessage`, `Posting to ${SIGN_MESSAGE_ENDPOINT}`)
-  // Cody TODO: Sign payload using account privkey
+  const body = JSON.stringify({
+    hashedPhoneNumber,
+    blindedQueryPhoneNumber: base64BlindedMessage.trim(),
+    account,
+  })
+
+  // Sign payload using account privkey
+  const authHeader = await contractKit.web3.eth.sign(body, account)
```

Comment:
Can you confirm this works when the app is in forno mode. You can enable it via the Settings page ('Data-saver mode')

Reply:
Verified that I can get the salt in data-saver mode:
![image](https://user-images.githubusercontent.com/11261382/81626408-75512200-93b0-11ea-85e6-b0538209fa0c.png)